src/listeners/errorRelated/errorHandling.ts

```diff
@@ -168,15 +163,7 @@ async function makeAndSendErrorEmbed<Options>(
 							)} properly for you. Please report this error ID to my developer: ${bold(inlineCode(errorUuid))}!`,
 						),
 					],
-					components: [
-						new ActionRowBuilder().setComponents([
-							new ButtonBuilder()
-								.setStyle(ButtonStyle.Link)
-								.setURL(supportServerInvite)
-								.setEmoji('🆘')
-								.setLabel('Support server'),
-						]),
-					],
+					components: [new ActionRowBuilder().setComponents([SupportServerButton])],
```

Comment:
In other places, I saw you call setComponents with just the button, not in an array